webtranslate/users/github.py

```diff
@@ -0,0 +1,134 @@
+"""
+User authentication and authorization via github teams.
+
+"""
+import configparser, os, json, requests, requests_oauthlib
+from webtranslate import rights, userauth
+
+
+
+github_organization = None
+github_org_api_token = None
+github_oauth_client_id = None
+github_oauth_client_secret = None
+
+
+
+def request_teams(name):
+    """
+    Request all teams an user is member of.
+
+    @param name: Github login
+    @type  name: C{str}
+
+    @return: Teams
+    @rtype:  C{set} of C{str}
+    """
+
+    try:
+        if github_organization is not None and github_org_api_token is not None:
```

Comment:
`if not github_organization or not github_org_api_token: return set()`

Return early, makes the flow of this function more readable. Means it can also move out of the try/except.